docs/en/api-reference/rpc_api.md

```diff
@@ -30,7 +30,7 @@ curl --request POST \
 You can also call these endpoints using **cleos** commands:
 
 ```
-cleos -u https://chain.wax.io get info
+cleos -u https://apiwax.3dkrender.com get info
```

Comment:
sure you want to add your public API to the main docs?

Reply:
Well... it's an example. And it is public. Maybe something like
`cleos -u [api-url] get info` ?

Reply:
Looking good!

```suggestion
cleos -u [api-url] get info
```